epochStart/metachain/validators.go

```diff
@@ -234,6 +247,34 @@ func (vic *validatorInfoCreator) VerifyValidatorInfoMiniBlocks(miniBlocks []*blo
 	return nil
 }
 
+func (vic *validatorInfoCreator) printAllMiniBlocks(created map[string]*block.MiniBlock, received []*block.MiniBlock) {
+	log.Debug("validatorInfoCreator.printAllMiniBlocks - printing created")
+	for _, mb := range created {
```

Comment:
maybe enforce the order (shard 0-meta)?

Reply:
actually, we have n miniblocks meta->all shards ID. Minor refactored the solution to keep the order.